### ISSUE TITLE
Sending MP3s shows 1:40 as length

### ISSUE THREAD

**Issue by Brawl345** [vysheng/tg#745]
When I send a MP3 through send_file, Telegram always shows "1:40" as length, but the mp3 is longer. This does not happen when I send it through a LUA script.

**Comment by LukeLR** [comment]
Are you sure your MP3-Files are correct? Sometimes they show less duration than they actually are. Check in a standard media player application.

**Comment by Brawl345** [comment]
Yes, all files I tested were ~3 minutes long.

**Comment by LukeLR** [comment]
And they show up as 3 minutes long in other applications?

**Comment by Brawl345** [comment]
Yes, even when I play them in Telegram Desktop (there is a blue bar over the group name, like a mini mp3 player)

**Comment by LukeLR** [comment]
Okay, then it's clearly not the MP3-File. Where do they show up as 1:40? In tg-cli?

**Comment by Brawl345** [comment]
In Telegram Desktop and in Telegram for Android.
See here:
![1](https://cloud.githubusercontent.com/assets/5049892/9357182/c0e18d54-4685-11e5-80e1-bf1097684efb.png)
The song is 03:32 long. When I send it through TDesktop, it shows the right size.

**Comment by LukeLR** [comment]
ah, okay, I Understand…

**Comment by jacboy** [comment]
Something new here?

**Comment by psycholyzern** [comment]
I have same issue. Uploaded a few seconds long mp3 file, but telegram showing 1:40.

**Comment by jegue** [comment]
@Brawl345 How do you send using  a LUA script?  I have the same problem shows 1:40 as length. Thanks

**Comment by dendendd** [comment]
Any updates on how to get around this? Thanks!
